src/dirbrowserformaction.cpp

```diff
@@ -106,13 +106,13 @@ void DirBrowserFormAction::process_operation(Operation op,
 						status);
 					f->set("listpos", "0");
 					std::string fn = utils::getcwd();
-					if (fn.back() != '/') {
-						fn.append(NEWSBEUTER_PATH_SEP);
+					if (fn.back() != NEWSBEUTER_PATH_SEP) {
+						fn.append(1, NEWSBEUTER_PATH_SEP);
```

Comment:
Wouldn't `fn.push_back(NEWSBEUTER_PATH_SEP)` be clearer?

Reply:
Yes, definitely.
I changed it for all occurrences of `std::string::append(1, ..)`.

By the way, let me know if ever you think I made some changes which you rather see in a separate pull request.

Reply:
> By the way, let me know if ever you think I made some changes which you rather see in a separate pull request.

Sure thing! These `append(1, x) -> push_back(x)` changes are too small to warrant a small PR though, it's fine to bundle them here.